detection_rules/devtools.py

```diff
@@ -836,20 +836,19 @@ def raw_permalink(raw_link):
 
     if update_coverage:
         coverage_file_path = get_path('docs', 'ATT&CK-coverage.md')
-        header_lines = textwrap.dedent("""
-        # Rule coverage
+        header_lines = textwrap.dedent("""# Rule coverage
 
-        ATT&CK navigator layer files are generated when a package is built with `make release` or
-        `python -m detection-rules`.This also means they can be downloaded from all successful builds.
+ATT&CK navigator layer files are generated when a package is built with `make release` or
+`python -m detection-rules`.This also means they can be downloaded from all successful builds.
 
-        These files can be used to pass to a custom navigator session. For convenience, the links are
-        generated below. You can also include multiple across tabs in a single session, though it is not
-        advisable to upload _all_ of them as it will likely overload your browsers resources.
+These files can be used to pass to a custom navigator session. For convenience, the links are
+generated below. You can also include multiple across tabs in a single session, though it is not
+advisable to upload _all_ of them as it will likely overload your browsers resources.
 
-        ## Current rule coverage
+## Current rule coverage
 
-        The source files for these links are regenerated with every successful merge to main. These represent
-        coverage from the state of rules in the `main` branch.
+The source files for these links are regenerated with every successful merge to main. These represent
+coverage from the state of rules in the `main` branch.
```

Comment:
Nit, technically the only change that is necessary to address the bug is the change on line(s) 839 and 840 -> 839 right?

Any particular reasoning to change this formatting here?

Reply:
With the removal of new line it added extra spaces like in this PR https://github.com/elastic/detection-rules/pull/4211 pinned in the testing section so we just reformatted so header section remains the same!

Reply:
![image](https://github.com/user-attachments/assets/fd5b980f-eada-4b96-b05a-0086d6199de2)

Reply:
I follow the logic, and think that makes since. Just for reference, it looks like that PR does not have a an extra spacing issue, just the initial newline issue? Perhaps its the wrong link or I am missing something?

![image](https://github.com/user-attachments/assets/e10eac47-86ec-4d3d-9789-20ec834f8a92)

Reply:
Yes, but when we removed the extra line, it caused more spacing issues like seen above and had to remove those spaces as well.

Reply:
Updated the summary with the right PR which caused spacing issues - https://github.com/elastic/detection-rules/pull/4264/files

Reply:
Appears like this might be the better link for that https://github.com/elastic/detection-rules/pull/4264/files :slightly_smiling_face: